keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,321 @@
+predictability.
+
+#### Reduce risk
+
+With higher predictability we can reduce the overall risk of changing the
+release schedule. The planning overhead of SIG Release gets reduced, while
+users of Kubernetes gain more time to adapt to the latest release.
+
+The current Kubernetes release cadence is so fast that most organizations
+cannot keep up with regularly making a minor version update every 3 months or
+going out of security support in a year. While releasing more frequently
+theoretically reduces the churn and risk of each release, this is only true if
+end users are actually able to apply the upgrades.
+
+#### Collecting data
+
+After this KEP is in place, SIG Release will follow up with a survey to collect
+feedback about the new release cadence.
+
+#### Creating a policy
+
+The outcome of this KEP is a policy for creating release schedules for
+Kubernetes.
+This allows the release team, as well as users, to follow a set of simple rules
+when it comes to knowing when and how Kubernetes releases will be scheduled.
+
+### Non-Goals
+
+#### Long-term support (LTS) releases
+
+The LTS Working Group was
+[disbanded](https://github.com/kubernetes/community/pull/5240) on October 20,
+2020.
+
+The outcome of their conversations was the proposal which established a
+[yearly support period][/keps/sig-release/1498-kubernetes-yearly-support-period/README.md]
+for minor releases of the project.
+
+While we may revisit the idea in the future, for now we trust the 2+ years of
+thoughtful deliberation by the working group enough to conclude that the
+project is not currently in a place to support long-term support releases.
+
+#### Changing enhancements graduation
+
+This KEP will not change the way that enhancements are being graduated. It's
+the responsibility of SIGs to keep track of their enhancements and graduate
+them in the provided constraints of SIG Architecture.
+
+The new release schedule will add room for only a few more weeks of
+development.
+SIGs should focus on using those additional weeks to enhance documentation and
+testing (stability)—not on adding more features. These decisions are not part
+of any SIG Release planning and will therefore be considered out of scope.
+
+#### Architecture changes
+
+Changing any architecture of Kubernetes—for example, decoupling its core
+components from the k/k repository—is outside the scope of this KEP.
+
+#### Modifying SIG Architecture policies
+
+Any policy change made by SIG Architecture is out of scope of this KEP. This
+non-goal corresponds partially to the [Changing enhancements
+graduation](#changing-enhancements-graduation) section.
+
+#### Accelerated release cycles
+
+The intent of this proposal is to create more opportunities to provide a
+high-value experience for Kubernetes consumers.
+
+The Kubernetes community faces a reasonable amount of tech debt across
+infrastructure, testing, policy, and documentation. This KEP proposes that we
+spend more time paying down that debt.
```

Comment:
I would love that, how can we be sure we achieve it?

Reply:
The release cycle needs to provide the right frame for the overall project. I think with this KEP we relax the time restrictions, which should result in follow-up discussions within the SIGs to use that time for tackling tech debt. In the end it will be only "some weeks more" for development, but maybe a SIG could use that time especially in the beginning of the cycle to work on this topic.

Let's follow-up together within the SIGs. 

(Feel free to reopen this discussion if that's not sufficient for you)